recipe/meta.yaml

```diff
@@ -50,6 +52,8 @@ outputs:
         - {{ pin_subpackage('pcmsolver', max_pin='x.x.x') }}
         - {{ pin_subpackage('libpcm', max_pin='x.x.x') }}
     requirements:
+      build:
+        - {{ stdlib("c") }}
```

Comment:
This was added overzealously by the piggyback migrator, and has been fixed in the meantime.
```suggestion
```